routes/api/users.js

```diff
@@ -106,8 +107,9 @@ router.post('/users', validate, (req, res, next) => {
       { sendMail: verificationHelper.sendVerificationEmail(user), user }))
     .then(({ user }) => {
       const payload = {
-        username: user.username,
+        id: user.id,
         email: user.email,
+        oauth: false,
```

Comment:
Since username is also created for OAuth users it would be good to include it back in the payload